dev-libs/libbase58/libbase58-0.1.4.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit autotools
+EAPI=7
```

Comment:
First of all please revbump with resetted KEYWORDS, this'd edit a stabilized ebuild in a dramatic way.